### ISSUE TITLE
Hash of Sitecore9.3.0 rev. 003498(WDP XM1 packages).zip is invalid

### ISSUE THREAD

**Issue by imranhamid83** [Sitecore/docker-images#320]
# ISSUE
I have cloned the latest sitecore image and ran the Build command on powershell. It processes for good 40 minutes and then chucks the error mentioned in the title. I've tried deleting image and ran the command but still the same issue. Are you able to help ? Tried to add the screenshot but not seeing any option to add the image to the issue.
Following is the error:

 Processing 3 of 17 'sitecore-assets:9.3.0-nanoserver-1809'...
Hash of 'Sitecore 9.3.0 rev. 003498 (WDP XM1 packages).zip' is invalid:
 Expected: EA323F9145BABFDBE6D292D505430AEBAC6B7DA4D904B5BCF71ACBC037CDC4CA
 Current : B39288F7C2CBDD1EBAAA969F7CA64D5C5123DE284DA23DE63DA98F9E6D08BD81

- [ y] I have tried with the latest version of Docker for Windows (Stable or Edge) or Docker Enterprise.
- [y ] Windows version is at least 1809 / ltsc2019.
- [y ] Windows has the latest updates installed.

## Expected behavior

...

## Actual behavior

...

## Additional information

- Output from `docker info`:

    ```text
    ...
    ```

- Are you running inside a virtualized Windows machine or physical:
- ...

## Steps to reproduce the behavior

  1. ...
  2. ...

**Comment by pbering** [comment]
Hi, this means that the download of that package is corrupted. You should do what the full error says: `"Hash of '{0}' is invalid:`n Expected: {1}`n Current : {2}`nThe target file '{3}' was deleted, please also check the source file '{4}' and see if it is corrupted, if so delete it and try again."`. Check the "Sitecore 9.3.0 rev. 003498 (WDP XM1 packages).zip" file in the InstallSourcePath (default .\packages). Delete it and it will be downloaded again.

**Comment by imranhamid83** [comment]
Thanks for your response.
As I mentioned above, I have tried deleting the file specified; it downloads it but then present with the same error unfortunately. 
I can delete the whole image and clone again to see if that works?

**Comment by imranhamid83** [comment]
![SharedScreenshot](https://user-images.githubusercontent.com/42209423/83616419-b8ea1680-a57f-11ea-997d-49ce5f12aa44.jpg)
![SharedScreenshot1](https://user-images.githubusercontent.com/42209423/83616562-e8991e80-a57f-11ea-9741-3714638f3966.jpg)

Add the two images above. One shows the file being downloaded and then it presents with the same error.
Thanks in advance.

**Comment by pbering** [comment]
can you try to download the package manually, check if you can unzip and put in ".\packages"?

**Comment by imranhamid83** [comment]
Do you mean to download the package from dev.sitecore.net? 
Even though I recently passed my certification I am not able to download anything. It presents me with the message saying login even though I am logged in and it shows my sitecore 9 certification in my profile(Things are never  straight forward with developers!)

**Comment by pbering** [comment]
yes, that is the same the script does with your credentials (**BTW you should change your password ASAP as it is visible in one of your screenshots**!)

**Comment by imranhamid83** [comment]
Thank You!
Never realised it... :(

**Comment by imranhamid83** [comment]
I am stuck at the moment as I am not able to download it from Sitecore dev site :( 
Any other solution?

**Comment by pbering** [comment]
Sorry, there is no other way if you don't have them already inside your organization.

**Comment by imranhamid83** [comment]
Thanks for your help. I will keep looking for a possible solution.

**Comment by MB-Hilo** [comment]
Changing version to 9.0.2 also doesn't resolve the issue

**Comment by imranhamid83** [comment]
@MB-Hilo 
Do you get the same error on the file specified above?

**Comment by MB-Hilo** [comment]
@imranhamid83 yes same file

**Comment by MB-Hilo** [comment]
@imranhamid83  you can get past this issue by commenting out lines 1870192 in docker-images\modules\SitecoreImageBuilder\1.0.0\Public\Invoke-Build.ps1

this disables the hash check...   which is not a good work around but still built for me

**Comment by pbering** [comment]
A less intrusive way is to skip hash validation with the `-SkipHashValidation` switch on `Invoke-Build`.

**Comment by MB-Hilo** [comment]
@pbering thanks 👍

**Comment by GenZodd** [comment]
I know this issue is closed but the issue was not fixed a workaround was just provided. Is there a plan to fix this issue so the workaround is not needed?
